app/chaturbate/echo.go

```diff
@@ -16,6 +17,7 @@ func newHub() *Hub {
 }
 
 type Hub struct {
+	sync.RWMutex
```

Comment:
add mutex as we can modify h.clients in different places in the same time to avoid race conditions